packages/blade/src/components/Input/DropdownInputTriggers/__tests__/__snapshots__/AutoComplete.native.test.tsx.snap

```diff
@@ -931,12 +931,12 @@ exports[`<Dropdown /> with <AutoComplete /> should render AutoComplete 1`] = `
                           <View
                             accessibilityElementsHidden={true}
                             data-blade-component="base-box"
-                            hasDescription={false}
```

Comment:
I wonder what this `hasDescription` as doing.

Reply:
nothing. It was stale lol. I guess I earlier added it for something but then removed its logic.